src/client/fuse_ll.cc

```diff
@@ -611,10 +611,34 @@ static void fuse_ll_read(fuse_req_t req, fuse_ino_t ino, size_t size, off_t off,
   CephFuse::Handle *cfuse = fuse_ll_req_prepare(req);
   Fh *fh = reinterpret_cast<Fh*>(fi->fh);
   bufferlist bl;
+#ifdef CEPH_HAVE_SPLICE
```

Comment:
where is CEPH_HAVE_SPLICE defined?

Reply:
I think we need to set   FUSE_CAP_SPLICE_{READ/WRITE} on fuse session's conn.want

Reply:
> I think we need to set FUSE_CAP_SPLICE_{READ/WRITE} on fuse session's conn.want

yeah, right. I will fix it. Thanks.